extensions/objects/models/objects.py

```diff
@@ -824,3 +824,23 @@ class ListOfSetsOfHtmlStrings(BaseObject):
         'type': 'list',
         'items': SetOfHtmlString.SCHEMA,
     }
+
+
+class DragAndDropHtmlString(BaseObject):
+    """A drag and drop Html string class."""
+
+    description = 'A drag and drop Html string.'
+    default_value = ''
+
+    SCHEMA = {
+        'type': 'html'
+    }
+
+
+class DragAndDropPositiveInt(BaseObject):
```

Comment:
Maybe, `DranAndDropRank`?

Reply:
I think previous one gives better insights about the type. It is similar to other objects.